### PR TITLE
(Step 6_1) Added required name field, made comments required.

### PR DESCRIPTION
**Pull request by cheoalee**
Added name as a feature of entity.

### CODE REVIEW

File: portfolio/src/main/webapp/data.html [hunk 1]
```diff
@@ -11,21 +11,21 @@
...
-      <textarea name="text-input" class="sansserif comment-page">Enter comment here...</textarea>
-      <br/><br/>
+      <label for="visitor-name" class="sansserif comment-page section-sep">Name:</label>
+      <input type="text" id="username" name="visitor-name" class="sansserif comment-page" required>
+
+      <label for="visitor-name" class="sansserif comment-page section-sep">Comment:</label>
+      <textarea name="visitor-comment" class="sansserif comment-page" required></textarea>
 
       <p class="sansserif comment-page">Add some spice:</p>
       <input type="checkbox" name="upper-case" class="sansserif" value="true"><label for="checkbox">Upper-case</label>
```
Comment (adamtcohen) [inline portfolio/src/main/webapp/data.html:22]: (pre-existing but noticing now) To correctly associate this label with your checkbox, I believe this should be the value of the id of your input.

File: portfolio/src/main/webapp/data.html [hunk 2]
```diff
@@ -11,21 +11,21 @@
...
     <h1 class="sansserif comment-page title">Comment Submission</h1>
-
     <form action="/data" method="POST">
-      <p class="sansserif comment-page section-sep">Leave a comment:</p>
-      <textarea name="text-input" class="sansserif comment-page">Enter comment here...</textarea>
-      <br/><br/>
+      <label for="visitor-name" class="sansserif comment-page section-sep">Name:</label>
+      <input type="text" id="username" name="visitor-name" class="sansserif comment-page" required>
+
+      <label for="visitor-name" class="sansserif comment-page section-sep">Comment:</label>
```
[the remark this answers is not in the document]
Comment (cheoalee) [reply portfolio/src/main/webapp/data.html]: Fixed (remove for attributes).

File: portfolio/src/main/webapp/data.html (hunk 1 above)
Comment (cheoalee) [reply portfolio/src/main/webapp/data.html:22]: Yep, I was mistakenly associating it with name. Thank you for the pointer!

File: portfolio/src/main/webapp/data.html [hunk 3]
```diff
@@ -11,21 +11,21 @@
...
-
     <form action="/data" method="POST">
-      <p class="sansserif comment-page section-sep">Leave a comment:</p>
-      <textarea name="text-input" class="sansserif comment-page">Enter comment here...</textarea>
-      <br/><br/>
+      <label class="sansserif comment-page section-sep">Name:</label>
+      <input type="text" id="username" name="visitor-name" class="sansserif comment-page" required>
+
+      <label class="sansserif comment-page section-sep">Comment:</label>
+      <textarea name="visitor-name" class="sansserif comment-page" required></textarea>
```
Comment (vidhyal) [inline portfolio/src/main/webapp/data.html:19]: This should still be visitor-comment right?

Comment (cheoalee) [reply portfolio/src/main/webapp/data.html]: Fixed! Thank you for the catch.

**Pull request merged**